docs/source/numpy.rst

```diff
@@ -378,3 +395,89 @@ xtensor universal functions are provided for a large set number of mathematical
 +-----------------------------------------------+-----------------------------------------------+
 | ``np.isfinite(a)``                            | ``xt::isfinite(a)``                           |
 +-----------------------------------------------+-----------------------------------------------+
+
+Linear Algebra Functions (xtensor-blas)
+---------------------------------------
+
+Many functions found in the ``numpy.linalg`` module are implemented in `xtensor-blas https://github.com/QuantStack/xtensor-blas`_,
```

Comment:
the syntax for the hyperlink is not correct. Examples here: https://raw.githubusercontent.com/QuantStack/xtensor-blas/master/docs/source/index.rst